tools/src/FileMetadata.cc

```diff
@@ -16,168 +16,162 @@
  * limitations under the License.
  */
 
+#include <getopt.h>
 #include <iostream>
 #include <fstream>
 #include <vector>
 #include <string>
 #include <sstream>
-#include <iomanip>
 
-#include "wrap/orc-proto-wrapper.hh"
 #include "orc/OrcFile.hh"
 
-using namespace orc::proto;
-
-uint64_t getTotalPaddingSize(const Footer& footer) {
-  uint64_t paddedBytes = 0;
-  StripeInformation stripe;
-  for (int stripeIx=1; stripeIx<footer.stripes_size(); stripeIx++) {
-      stripe = footer.stripes(stripeIx-1);
-      uint64_t prevStripeOffset = stripe.offset();
-      uint64_t prevStripeLen = stripe.datalength() + stripe.indexlength() +
-        stripe.footerlength();
-      paddedBytes += footer.stripes(stripeIx).offset() -
-        (prevStripeOffset + prevStripeLen);
-  };
-  return paddedBytes;
+void printStripeInformation(std::ostream& out,
+                            uint64_t index,
+                            uint64_t columns,
+                            std::unique_ptr<orc::StripeInformation> stripe,
+                            bool verbose) {
+  out << "    { \"stripe\": " << index
+      << ", \"rows\": " << stripe->getNumberOfRows() << ",\n";
+  out << "      \"offset\": " << stripe->getOffset()
+      << ", \"length\": " << stripe->getLength() << ",\n";
```

Comment:
The lines get really long and hard to read. I don't think we should merge those lines. I've grouped related fields together on to the same line.